li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerImpl.java

```diff
@@ -689,10 +717,12 @@ private void handleLiClosestResetStrategy(InvalidOffsetException oe) {
 
       if (!seekBeginningPartitions.isEmpty()) {
         LOG.info("Offsets are out of range for partitions {}. Seeking to the beginning offsets returned", seekBeginningPartitions);
+        _offsetInvalidOrOutRangeCount.getAndIncrement();
```

Comment:
Why is the counter increment here as well?
I think it is worth clarifying what count we are referring to with this metrics - is it the number of times the underlying ("apache") client is throwing these exceptions? Or something else as well?

Reply:
Yes, this metrics counts the total number of times the apache consumer clients throwing the exceptions of "OffsetOutOfRange", which marks the potential data loss